UefiCpuPkg/Library/MtrrLib/UnitTest/MtrrLibUnitTest.c

```diff
@@ -30,6 +30,8 @@ STATIC MTRR_LIB_SYSTEM_PARAMETER  mSystemParameters[] = {
   { 48, TRUE, TRUE, CacheWriteThrough,   12 },
   { 48, TRUE, TRUE, CacheWriteProtected, 12 },
   { 48, TRUE, TRUE, CacheWriteCombining, 12 },
+
+  { 48, TRUE, TRUE, CacheWriteBack,      12, 7}, // 7 bits for MKTME

```

Comment:
Shouldn't that be as below?
`{ 48, TRUE, TRUE, CacheWriteBack, 7}, // 7 bits for MKTME`
Otherwise UTs won't build